src/Pages/Withdrawals.razor

```diff
@@ -581,11 +579,36 @@
         {
             ToastService.ShowError("Something went wrong");
             _userPendingRequests.Remove(arg.Item);
-
         }
+    }
+
+private string ShowRequestor(WalletWithdrawalRequest walletWithdrawalRequest)
+{
+    if (walletWithdrawalRequest.RequestMetadata != null)
+    {
+        var requestMetadata = JsonConvert.DeserializeObject<Dictionary<string, string>>(walletWithdrawalRequest.RequestMetadata);
```

Comment:
What happens if the request metadata is not a json string, for example a plain text?

Reply:
Make sense to wrap try-catch block. Btw we can manage the type of metadata.